app/views/directives/notifications/notification-counter.html

```diff
@@ -0,0 +1,13 @@
+<a
+  ng-if="!$ctrl.hide"
+  href=""
+  ng-click="$ctrl.onClick()"
+  class="nav-item-iconic relative">
+  <div
+    ng-if="$ctrl.showNewNotificationIndicator"
+    class="notification-counter notification-counter-offset"
+    row main-axis="center" cross-axis="center">
+    <span><!--{{countDisplay}}--></span>
+  </div>
+  <span class="fa fa-bell" aria-hidden="true"></span>
```

Comment:
This strays from the patternfly designs adding indicator colors and such. Why not follow the example from patternfly:


```
<li class="drawer-pf-trigger">
  <a class="nav-item-iconic">
    <span class="fa fa-bell" title="Notifications"></span>
    <span class="badge"> </span>
  </a>
</li>

```